packages/atomic/cypress/integration/facets/color-facet/color-facet.cypress.ts

```diff
@@ -566,4 +569,124 @@ describe('Color Facet Test Suites', () => {
       });
     });
   });
+
+  describe('with depends-on', () => {
+    const facetId = 'abc';
+    describe('as a dependent', () => {
+      const parentFacetId = 'def';
+      const parentField = 'author';
+      const expectedValue = 'BPA';
+      before(() => {
+        new TestFixture()
+          .with(
+            addColorFacet({
+              'facet-id': facetId,
+              field: colorFacetField,
+              [`depends-on-${parentFacetId}`]: expectedValue,
+            })
+          )
+          .with(addFacet({'facet-id': parentFacetId, field: parentField}))
+          .init();
+      });
+
+      CommonFacetAssertions.assertDisplayFacet(
+        ColorFacetSelectors.withId(facetId),
+        false
+      );
+      CommonFacetAssertions.assertDisplayFacet(
+        FacetSelectors.withId(parentFacetId),
+        true
+      );
+
+      describe('when the dependency is met', () => {
+        before(() => {
+          typeFacetSearchQuery(
+            FacetSelectors.withId(parentFacetId),
+            expectedValue,
+            true
+          );
+          selectIdleCheckboxValueAt(FacetSelectors.withId(parentFacetId), 0);
+        });
+
+        CommonFacetAssertions.assertDisplayFacet(
+          ColorFacetSelectors.withId(facetId),
+          true
+        );
+        CommonFacetAssertions.assertDisplayFacet(
+          FacetSelectors.withId(parentFacetId),
+          true
+        );
+      });
+    });
+
+    describe('as a parent', () => {
+      const dependentFacetId = 'def';
+      const dependentField = 'author';
+      const expectedValue = 'doc';
+      before(() => {
+        new TestFixture()
+          .with(addColorFacet({'facet-id': facetId, field: colorFacetField}))
+          .with(
+            addFacet({
+              'facet-id': dependentFacetId,
+              field: dependentField,
+              [`depends-on-${facetId}`]: expectedValue,
+            })
+          )
+          .init();
+      });
+
+      CommonFacetAssertions.assertDisplayFacet(
+        FacetSelectors.withId(dependentFacetId),
+        false
+      );
+      CommonFacetAssertions.assertDisplayFacet(
+        ColorFacetSelectors.withId(facetId),
+        true
+      );
+
+      describe('when the dependency is met', () => {
+        before(() => {
+          typeFacetSearchQuery(
+            ColorFacetSelectors.withId(facetId),
+            expectedValue,
+            true
+          );
+          selectIdleBoxValueAt(0);
+        });
+
+        CommonFacetAssertions.assertDisplayFacet(
+          FacetSelectors.withId(dependentFacetId),
+          true
+        );
+        CommonFacetAssertions.assertDisplayFacet(
+          ColorFacetSelectors.withId(facetId),
+          true
+        );
+      });
+    });
+
+    describe('with two dependencies', () => {
```

Comment:
I think testing for multiple level of dependency is also important

Reply:
I forgot to include in the list to review that I added `facet-depends-on.cypress.ts`, which covers multiple level.